openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/ActivityFeed.spec.ts

```diff
@@ -948,39 +948,52 @@ base.describe('Activity feed with Data Consumer User', () => {
         assignee: viewAllUser.responseData.name,
       };
 
-      await base.step('Create and Assign Task to user 3', async () => {
-        await redirectToHomePage(page1);
-        await entity3.visitEntityPage(page1);
+      try {
+        await base.step('Create and Assign Task to user 3', async () => {
+          await redirectToHomePage(page1);
+          await entity3.visitEntityPage(page1);
 
-        await page1.getByTestId('request-description').click();
+          await page1.getByTestId('request-description').click();
 
-        await createDescriptionTask(page1, value);
+          await createDescriptionTask(page1, value);
 
-        await afterActionUser1();
-      });
+          await afterActionUser1();
+        });
 
-      await base.step(
-        'Accept Task By user 2 should throw error for since it has only viewAll permission',
-        async () => {
-          await redirectToHomePage(page2);
+        await base.step(
+          'Accept Task By user 2 should throw error for since it has only viewAll permission',
+          async () => {
+            await redirectToHomePage(page2);
 
-          await entity3.visitEntityPage(page2);
+            await entity3.visitEntityPage(page2);
 
-          await page2.getByTestId('activity_feed').click();
+            await page2.getByTestId('activity_feed').click();
 
-          await page2.getByRole('menuitem', { name: 'Tasks' }).click();
+            const taskResponse = page2.waitForResponse(
+              '/api/v1/feed?entityLink=**type=Task&taskStatus=Open'
+            );
+            await page2.getByRole('menuitem', { name: 'Tasks' }).click();
+            await taskResponse;
 
-          await page2.getByText('Accept Suggestion').click();
+            await page2.getByText('Accept Suggestion').click();
 
-          await toastNotification(
-            page2,
-            // eslint-disable-next-line max-len
-            `Principal: CatalogPrincipal{name='${viewAllUser.responseData.name}'} operation EditDescription denied by role ${viewAllRoles.responseData.name}, policy ${viewAllPolicy.responseData.name}, rule editNotAllowed`
-          );
+            await toastNotification(
+              page2,
+              // eslint-disable-next-line max-len
+              `Principal: CatalogPrincipal{name='${viewAllUser.responseData.name}'} operation EditDescription denied by role ${viewAllRoles.responseData.name}, policy ${viewAllPolicy.responseData.name}, rule editNotAllowed`
+            );
```

Comment:
if the user doesn't have permission, we should hide the button, right?

Reply:
yes, we shouldn't. We will cover this case separately as we need to add a permission check and test it thoroughly. With another policy user as well.